presto-main/src/main/java/io/prestosql/server/PluginClassLoader.java

```diff
@@ -141,21 +139,4 @@ private static String classNameToResource(String className)
     {
         return className.replace('.', '/');
     }
-
-    @SuppressWarnings("JavaReflectionMemberAccess")
-    private static ClassLoader findPlatformClassLoader()
-    {
-        try {
-            // use platform class loader on Java 9
-            Method method = ClassLoader.class.getMethod("getPlatformClassLoader");
-            return (ClassLoader) method.invoke(null);
-        }
-        catch (NoSuchMethodException ignored) {
-            // use null class loader on Java 8
-            return null;
```

Comment:
This is the line that only be fixed. `null` (bootstrap CL) should be replaced with `getSystemClassLoader().getParent())`.

Reply:
As I stated above, our goal is not to get the platform cl, but instead "The goal with the plugin class loader is to exclude the system classes, because these are the Presto application classes."  This means we want to use the same code for all JVM versions.